app/vulncode_app/views.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.http import HttpResponse
+from django.shortcuts import render
+
+import vulncode_app.api_data as api
+
+def product(request, product):
+    """
+    Queries the cve-search api with just
+    a product name.
+    """
+    data = api.output_cve_id(name=product)
+    return HttpResponse(data)
+
+def product_ver(request, product, ver):
```

Comment:
Why using `version` in the api module and `ver` here?
Always prefer to use easy to read and unambiguous variable name.
Also be consistent with your naming within the codebase,  if the content is the same, use the same name.
`def product_ver(request, product, ver):`
->
`def product_version(request, product, version):`